File: typescript/base-hmi-types/src/screens/widgets/HmiArrowIndicator.ts
import { HmiObjectType } from "../base/HmiObjectType.js";
import { HmiScaleWidgetBase } from "./HmiScaleWidgetBase.js";

/** A value-driven arrow that moves along a horizontal or vertical scale. */
export class HmiArrowIndicator extends HmiScaleWidgetBase {
  constructor() {
    super();
    this.hmiObjectType = HmiObjectType.HmiArrowIndicator;
  }
}

File: src/BaseHmiTypes/Screens/Widgets/HmiArrowIndicator.cs
using BaseHmiTypes.Screens.Base;

namespace BaseHmiTypes.Screens.Widgets;

/// <summary>
/// A value-driven arrow that moves along a horizontal or vertical scale.
/// </summary>
public class HmiArrowIndicator : HmiScaleWidgetBase
{
    public HmiArrowIndicator()
    {
        HmiObjectType = BaseHmiTypes.Screens.Base.HmiObjectType.HmiArrowIndicator;
    }
}
